lib/Day10Year2023/MapsBuilder.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Lib.Day05Year2023;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lib.Day10Year2023
+{
+    public class MapsBuilder
+    {
+        public Pipe GetStartPipe(string[] input) {
```

Comment:
Cette méthode fait plusieurs choses sans le dire
```suggestion
        public Pipe CreateNetworkAndGetStartPipe(string[] input) {
```